sdk/monitor/monitor-query/README.md

```diff
@@ -541,7 +541,7 @@ QueryMetricsResult
 
 ```ts
 import { DefaultAzureCredential } from "@azure/identity";
-import { Durations, Metric, MetricsQueryClient } from "@azure/monitor-query";
+import { Durations, Metric, MetricsQueryClient } from "../src/index";
```

Comment:
Any reason why we wouldn't want `@azure/monitor-query` here?

Reply:
Umm typo. Was testing intellisense locally.
```suggestion
import { Durations, Metric, MetricsQueryClient } from "@azure/monitor-query";
```